--- LYON_7EME_BATI_2009.gml	2014-12-18 09:44:28.000000000 +0100
+++ LYON_7EME_BATI_2009.gml.patched	2020-01-20 13:07:22.000000000 +0100
@@ -3065363,33 +3065363,6 @@
     </bldg:Building>
   </cityObjectMember>
 
-  <cityObjectMember>
-    <bldg:Building gml:id="LYON_7EME_00110">
-      <gen:doubleAttribute name="Volume">
-        <gen:value>0.000</gen:value>
-      </gen:doubleAttribute>
-      <bldg:boundedBy>
-        <bldg:RoofSurface gml:id="UUID_cd32dd29-39ea-42a7-8960-24f8c4337b2a">
-          <gen:doubleAttribute name="Area">
-            <gen:value>0.000</gen:value>
-          </gen:doubleAttribute>
-          <bldg:lod2MultiSurface>
-            <gml:MultiSurface srsDimension="3"/>
-          </bldg:lod2MultiSurface>
-        </bldg:RoofSurface>
-      </bldg:boundedBy>
-      <bldg:boundedBy>
-        <bldg:WallSurface gml:id="UUID_5ba52321-2bc0-4fec-80a2-4d4e6c5be367">
-          <gen:doubleAttribute name="Area">
-            <gen:value>0.000</gen:value>
-          </gen:doubleAttribute>
-          <bldg:lod2MultiSurface>
-            <gml:MultiSurface srsDimension="3"/>
-          </bldg:lod2MultiSurface>
-        </bldg:WallSurface>
-      </bldg:boundedBy>
-    </bldg:Building>
-  </cityObjectMember>
 
   <gml:boundedBy>
     <gml:Envelope srsDimension="3" srsName="EPSG:3946">
