--- LYON_8EME_BATI_2012.gml	2014-12-18 09:41:20.000000000 +0100
+++ LYON_8EME_BATI_2012.gml.patched	2020-01-22 15:57:21.000000000 +0100
@@ -1099144,7 +1099144,7 @@
           </app:target>
           <app:target uri="#UUID_8bc09be3-5c8b-4124-a6bd-8a3a01413afb">
             <app:TexCoordList>
-              <app:textureCoordinates ring="#UUID_cae90897-97ba-4e86-9d8a-c46ec2823b37">0.352869 -1.#IND00 0.353114 -1.#IND00 0.353111 -1.#IND00 0.352869 -1.#IND00 </app:textureCoordinates>
+              <app:textureCoordinates ring="#UUID_cae90897-97ba-4e86-9d8a-c46ec2823b37">0.352869 -1.00 0.353114 -1.00 0.353111 -1.00 0.352869 -1.00 </app:textureCoordinates>
             </app:TexCoordList>
           </app:target>
           <app:target uri="#UUID_c16d2e35-6708-493c-ad89-04135a0e13e3">
