--- LYON_7EME_BATI_2012.gml	2014-12-18 09:38:46.000000000 +0100
+++ LYON_7EME_BATI_2012.gml.patched	2020-01-22 15:49:40.000000000 +0100
@@ -2752387,33 +2752387,6 @@
     </bldg:Building>
   </cityObjectMember>
 
-  <cityObjectMember>
-    <bldg:Building gml:id="LYON_7EME_00215">
-      <gen:doubleAttribute name="Volume">
-        <gen:value>0.000</gen:value>
-      </gen:doubleAttribute>
-      <bldg:boundedBy>
-        <bldg:RoofSurface gml:id="UUID_bf7281a8-8c3a-4952-ba18-ba29b844640b">
-          <gen:doubleAttribute name="Area">
-            <gen:value>0.000</gen:value>
-          </gen:doubleAttribute>
-          <bldg:lod2MultiSurface>
-            <gml:MultiSurface srsDimension="3"/>
-          </bldg:lod2MultiSurface>
-        </bldg:RoofSurface>
-      </bldg:boundedBy>
-      <bldg:boundedBy>
-        <bldg:WallSurface gml:id="UUID_f4174014-e491-4ef7-99a5-9c2c8caccef0">
-          <gen:doubleAttribute name="Area">
-            <gen:value>0.000</gen:value>
-          </gen:doubleAttribute>
-          <bldg:lod2MultiSurface>
-            <gml:MultiSurface srsDimension="3"/>
-          </bldg:lod2MultiSurface>
-        </bldg:WallSurface>
-      </bldg:boundedBy>
-    </bldg:Building>
-  </cityObjectMember>
 
   <cityObjectMember>
     <bldg:Building gml:id="LYON_7EME_00038">
